megamek/src/megamek/server/Server.java

```diff
@@ -10802,8 +10802,10 @@ public void deliverMortarFlare(Coords coords, int duration) {
      */
 
     public void deliverMissileSmoke(Coords coords, int smokeType, Vector<Report> vPhaseReport) {
-        Report r = new Report(5185, Report.PUBLIC);
+        Report r = new Report(5183, Report.PUBLIC);
         r.indent(2);
+        //Report either light or heavy smoke, as appropriate
+        r.choose(smokeType == 1);
```

Comment:
Use the named constant instead of a literal.

Reply:
Oops. Fixed.